bin/rector.php

```diff
@@ -27,7 +27,7 @@
 $autoloadIncluder->includeDependencyOrRepositoryVendorAutoloadIfExists();
 
 $autoloadIncluder->loadIfExistsAndNotLoadedYet(__DIR__ . '/../vendor/scoper-autoload.php');
-$autoloadIncluder->loadIfExistsAndNotLoadedYet(getcwd() . '/vendor/autoload.php');
+$autoloadIncluder->loadIfExistsAndNotLoadedYet(__DIR__ . '/../vendor/autoload.php');
```

Comment:
I think add new call is safer instead of change existing value:

```suggestion
$autoloadIncluder->loadIfExistsAndNotLoadedYet(getcwd() . '/vendor/autoload.php');
$autoloadIncluder->loadIfExistsAndNotLoadedYet(__DIR__ . '/../vendor/autoload.php');
```

Reply:
No, I think we have to _remove_ the `cwd()`-based one.

Reply:
that  will broke when using `composer require rector/rector:dev-patch-1` with pointed to your repository. I tried your PR branch, and it got error:

```bash
PHP Fatal error:  Uncaught Error: Class "Rector\Core\Console\Style\SymfonyStyleFactory" not found in /Users/samsonasik/www/CodeIgniter4/vendor/rector/rector/bin/rector.php:35
Stack trace:
#0 /Users/samsonasik/www/CodeIgniter4/vendor/rector/rector/bin/rector(4): require_once()
#1 {main}
  thrown in /Users/samsonasik/www/CodeIgniter4/vendor/rector/rector/bin/rector.php on line 35
Fatal error: Uncaught Error: Class "Rector\Core\Console\Style\SymfonyStyleFactory" not found in /Users/samsonasik/www/CodeIgniter4/vendor/rector/rector/bin/rector.php:35
Stack trace:
#0 /Users/samsonasik/www/CodeIgniter4/vendor/rector/rector/bin/rector(4): require_once()
#1 {main}
  thrown in /Users/samsonasik/www/CodeIgniter4/vendor/rector/rector/bin/rector.php on line 35
```

Reply:
Oh, I see... in Composer-based setups, it would have to be `__DIR__.'/../../vendor/autoload.php` probably?

Reply:
@samsonasik Does it work if you completely remove the line in question here? So both ones, with `cwd()` and `__DIR__`?

Reply:
should be : 

```php
__DIR__ . '/../../../../vendor/autoload.php'
```

You can check possible vendor location from : 

https://github.com/rectorphp/rector/blob/master/src/Stubs/PHPStanStubLoader.php#L18-L23

that came from `src/Stubs` so, the upping can be reduced 1 per-each from `bin` directory.

Reply:
Not `../../../`? (see the latest commit I just pushed)

Reply:
Not sure about https://github.com/rectorphp/rector/blob/master/src/Stubs/PHPStanStubLoader.php#L18-L23, but maybe that needs a bit of scrutinity, too...

Reply:
`__DIR__ . '/../../../../vendor/autoload.php'` is same with `__DIR__ . '/../../../autoload.php'`. Just add more "up" to ensure the parent is named "vendor"

Reply:
Does my latest commit work?

Reply:
yes, that seems works.